pkg/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -173,6 +178,31 @@ func getIgnitionConfig(assetsDir string, profile *performancev1alpha1.Performanc
 	}
 
 	nonIsolatedCpus := profile.Spec.CPU.NonIsolated
+
+	err := ioutil.WriteFile(fmt.Sprintf("%s/setAffinity.conf",assetsDir), []byte("[Manager]\nCPUAffinity="+string(*nonIsolatedCpus)), 0700)
```

Comment:
do you need this write?
Can you:
- read file
- change content to include right CPU affinity
- encode content and provide it to the machine-config

Reply:
@cynepco3hahue  this file does not exist.  so i write this file created with the righ cpu affinity list. I dont see much difference from modifying an existing file oppose to creating one (its not runtime anyway). one question though.. do i need to make sure this file does not exist in the dir im writing it to (assets) before writing it ?

Reply:
I missed the fact that the file does not exist, I thought it already exists under the assets dir.
If the file exists the `WriteFile` will truncate it, see - https://golang.org/pkg/io/ioutil/#WriteFile

Why do you need to write it?

Just use:
```go
contentBase64 := base64.StdEncoding.EncodeToString([]byte("[Manager]\nCPUAffinity="+string(*nonIsolatedCpus)))
```